.github/workflows/push.yaml

```diff
@@ -0,0 +1,32 @@
+name: ci
+
+on:
+  push:
+    branches: main
+
+jobs:
+  build_container:
+    runs-on: ubuntu-18.04 # for older version of awscli.
+    steps:
+      - id: file_changes
+        uses: trilom/file-changes-action@v1.2.3
+
+      - name: Checkout
+        if: contains(toJSON(steps.file_changes.outputs.files), 'infra/')
```

Comment:
**Question**: Do we now still need this condition? We checkout no matter the change is in `infra/` or not.

Reply:
Hm, thanks!  You're right.